### PR TITLE
[#2991] [#2988] docs and grid updates

### PR DESCRIPTION
**Pull request by aluarius**
Closes: #2991 
Closes: #2988

## Description
- [x] restored grid deprecated alerts;
- [x] all grid examples are functional;
- [x] smart helpers moved out from grid;
- [x] smart helpers rewritten without deprecated grid helpers.   
</details>

## Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [x] Improvement/refactoring (non-breaking change that doesn't add any feature but make things better)

### CODE REVIEW

File: packages/docs/page-config/styles/smart-helpers/examples/Spacing.vue [hunk 1]
```diff
@@ -1,41 +1,41 @@
 <template>
```
Comment (rustem-nasyrov) [inline packages/docs/page-config/styles/smart-helpers/examples/Spacing.vue:1]: Seems like doesn't work. 
image

**Review by rustem-nasyrov — CHANGES_REQUESTED** [review]
image
Maybe better to paint it red?

File: packages/docs/page-config/styles/smart-helpers/examples/Spacing.vue (hunk 1 above)
Comment (m0ksem) [reply packages/docs/page-config/styles/smart-helpers/examples/Spacing.vue:1]: @aluarius, make sure plugins are registered in nuxt module.

**Review by rustem-nasyrov — APPROVED** [review]

**Review by m0ksem — APPROVED** [review]
Perfect.

**Pull request merged**